sonic-pcied/scripts/pcied

```diff
@@ -41,8 +41,7 @@ class DaemonPcied(DaemonBase):
     def __init__(self, log_identifier):
         super(DaemonPcied, self).__init__(log_identifier)
 
-        platform = get_platform()
-        pciefilePath = os.path.join(PLATFORM_ROOT_PATH, platform, "plugins", PCIE_CONF_FILE)
+        pciefilePath = os.path.join(PLATFORM_ROOT_PATH, "plugins", PCIE_CONF_FILE)
```

Comment:
As a proper fix, we should remove the `PLATFORM_ROOT_PATH` and replace with a call to `get_paths_to_platform_and_hwsku_dirs()` in the new sonic-py-common package. Please see the following for an example: https://github.com/Azure/sonic-platform-daemons/blob/master/sonic-xcvrd/scripts/xcvrd#L1086

Reply:
Fixed, please review.